src/python/grpcio/grpc/experimental/aio/_call.py

```diff
@@ -270,8 +275,12 @@ def __await__(self) -> ResponseType:
             if self._cython_call.is_locally_cancelled():
                 raise asyncio.CancelledError()
             else:
+                call_status = self._cython_call._status
+                debug_error_string = None
+                if call_status is not None:
+                    debug_error_string = call_status._debug_error_string
```

Comment:
I would say that we have public methods for accessing to the attributes that you are using:

* `status` can be retrieved by [1], its a coroutine so if you want to make sure that the RPC is in a finished state you can always use [2]
* `debug_error_string` can be retrieved through [3]

[1] https://github.com/grpc/grpc/blob/master/src/python/grpcio/grpc/_cython/_cygrpc/aio/call.pyx.pxi#L232
[2] https://github.com/grpc/grpc/blob/master/src/python/grpcio/grpc/_cython/_cygrpc/aio/call.pyx.pxi#L210
[3] https://github.com/grpc/grpc/blob/master/src/python/grpcio/grpc/_cython/_cygrpc/aio/rpc_status.pxd.pxi#L28